static/app/views/replays/detail/focusTabs.tsx

```diff
@@ -16,23 +17,35 @@ const ReplayTabs: Record<TabKey, string> = {
 type Props = {};
 
 function FocusTabs({}: Props) {
+  const {location} = useRouteContext();
   const {getActiveTab, setActiveTab} = useActiveReplayTab();
   const activeTab = getActiveTab();
+
   return (
     <NavTabs underlined>
-      {Object.entries(ReplayTabs).map(([tab, label]) => (
-        <li key={tab} className={activeTab === tab ? 'active' : ''}>
-          <a
-            href={`#${tab}`}
-            onClick={(e: MouseEvent) => {
-              setActiveTab(tab);
-              e.preventDefault();
-            }}
-          >
-            <span>{label}</span>
-          </a>
-        </li>
-      ))}
+      {Object.entries(ReplayTabs).map(([tab, label]) => {
+        const href = Object.entries({
+          ...location.query,
+          t_main: tab,
+        }).reduce(
+          (acc, [queryParam, val], i) => `${acc}${i > 0 ? '&' : ''}${queryParam}=${val}`,
+          `${location.pathname}?`
+        );
```

Comment:
This is a bit hard to grok -- I'm assuming we're trying to preserve the query string here... can we:

1) Add comments
2) Maybe use the `qs` (query-string?) library?

Reply:
Much nicer.. I should have checked for a lib from the start